ext/snmp/snmp.stub.php

```diff
@@ -23,6 +23,8 @@ function snmp_set_enum_print(bool $enable): bool {}
 
 function snmp_set_oid_output_format(int $format): bool {}
 
+function snmp_set_string_output_format(int $format): bool {}
```

Comment:
Okay, now I at least see why you picked the bool return value... I would still go for `: void` here, as the `: bool` is really a backwards-compatibility leftover from a time where these functions could return both true and false.

Reply:
Taking your word for it, I'm the old new guy ;)